packages/Adapters/BasicGeometry/src/DTK_BasicEntitySet.hpp

```diff
@@ -92,9 +93,11 @@ class BasicEntitySetIterator : public EntityIterator
     Entity *operator->(void)override;
 
     // Equal comparison operator.
+    IGNORE_UNDEFINED_SANITIZE
```

Comment:
We need to make sure we have a policy of using this macro only for code that is deprecated. Any new code or code that will remain in the repo should not have this macro.

Per our previous discussions this use case is a good example of when to use the macro as the old interfaces are going to be removed.